examples/02-mesh/plot_quadtree_facediv.py

```diff
@@ -4,18 +4,25 @@
 
 Showing the face divergence on the quadtree with numbering.
 """
-import numpy as np
 from SimPEG import Mesh
 import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
It is slightly better practice to import standard things first, then more custom after, e.g. first import numpy, matplotlib then SimPEG studd